index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./style.css" />
+    <title>PortfolioPage</title>
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+  </head>
+  <body>
```

Comment:
- [ ] Please fix the margins and padding here and make it look exactly like Figma design try to change the overflow here

![image](https://user-images.githubusercontent.com/74744787/164726708-bbe3e4b1-5c4a-4524-8ad3-9d506e5030cd.png)